File: 8-kyu/did-she-say-hallo/did-she-say-hallo.py
def validate_hello(greetings):
    #O(n)
    cumprimentos = ['hello', 'ciao', 'salut', 'hallo', 'hola', 'ahoj', 'czesc']
    for palavra in cumprimentos:
        if palavra in greetings.lower():
            return True
    else:
        return False

File: 8-kyu/did-she-say-hallo/did-she-say-hallo.java
public class Kata{
  public static boolean validateHello(String greetings){
    greetings = greetings.toLowerCase();
    String[] cumprimentos = {"hello", "ciao", "salut", "hallo", "hola", "ahoj", "czesc"};
    for (int i =0; i < 7; i++){
        if(greetings.contains(cumprimentos[i])) return true;
        }
    return false;
  }
  
}